.github/actions/push-container/action.yml

```diff
@@ -0,0 +1,32 @@
+name: Push Container
+description: Push a Container
+
+inputs:
+  registry:
+    description: Server address of Docker registry.
+    required: true
+  username:
+    description: Username used to login.
+    required: true
+  password:
+    description: Password or PAT used to login.
+    required: true
+
+  name:
+    description: The name of the image to push.
+    required: true
+  tag:
+    description: The tag of the image to push.
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - uses: docker/login-action@v2
+      with:
+        registry: ${{ inputs.registry }}
+        username: ${{ inputs.username }}
+        password: ${{ inputs.password }}
+
+    - run: |
```

Comment:
It seems like this will need a `shell` property:

```
- shell: bash
  run |
...
```

I ran into this while deduplicating the build workflow:

https://github.com/nathanchance/containers/actions/runs/2402420380